vcd/resource_vcd_org_vdc_test.go

```diff
@@ -122,13 +122,30 @@ func runOrgVdcTest(t *testing.T, params StringMap, allocationModel string) {
 	}
 	debugPrintf("#[DEBUG] CONFIGURATION: %s", configText)
 
+	usedNetworkPool := testConfig.VCD.ProviderVdc.NetworkPool
+	expectNonEmptyPlan := false
+	// in vCD version 10 with NXT - network pool reference isn't returned with AdminVdc
+	vcdClient, err := getTestVCDFromJson(testConfig)
+	if err != nil {
+		t.Error(fmt.Errorf("error getting client configuration: %s", err))
+	}
+	err = vcdClient.Authenticate(testConfig.Provider.User, testConfig.Provider.Password, testConfig.Provider.SysOrg)
+	if err != nil {
+		t.Error(fmt.Errorf("authentication error: %s", err))
+	}
+	if vcdClient.APIVCDMaxVersionIs(">= 33.0") {
+		usedNetworkPool = ""
+		expectNonEmptyPlan = true
+	}
+
 	resource.Test(t, resource.TestCase{
 		PreCheck:     func() { testAccPreCheck(t) },
 		Providers:    testAccProviders,
 		CheckDestroy: testAccCheckVdcDestroy,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config: configText,
+				Config:             configText,
+				ExpectNonEmptyPlan: expectNonEmptyPlan,
```

Comment:
If you have a suppress in code - why is this needed? The non empty plan signals that it will cause an "update" on every `apply`, isn't it so?

Reply:
this testing framework issue, I tested in real life and supressFunc works, but in testing case it sees that network pool is empty and provided one isn't.